.eslintrc.cjs

```diff
@@ -629,31 +631,18 @@ const rules = {
   '@typescript-eslint/strict-boolean-expressions': 0,
   '@typescript-eslint/switch-exhaustiveness-check': 1,
   '@typescript-eslint/triple-slash-reference': [1, { path: 'never', types: 'never', lib: 'never' }],
-  '@typescript-eslint/type-annotation-spacing': 0,
   '@typescript-eslint/typedef': 0,
   '@typescript-eslint/unbound-method': 0,
   '@typescript-eslint/unified-signatures': 0,
 
   // @typescript-eslint/eslint-plugin Extension Rules
   // https://typescript-eslint.io/rules/#extension-rules
-  '@typescript-eslint/block-spacing': 0,
```

Comment:
https://typescript-eslint.io/blog/deprecating-formatting-rules/#upgrading-to-eslint-stylistic